dataproc/src/test/java/CreateClusterTest.java

```diff
@@ -75,8 +76,13 @@ public void createClusterTest() throws IOException, InterruptedException {
 
   @After
   public void tearDown() throws IOException, InterruptedException {
+    String myEndpoint = String.format("%s-dataproc.googleapis.com:443", REGION);
```

Comment:
Is there a way to file an issue on the client library for this? 

The region is already passed in the `deleteClusterAsyncRequest`, and it seems pretty dumb to force the user to create a new regional client just to delete a cluster.

Reply:
I agree and this is consistent across all of the languages. I have this documented on my end.